src/pkg/transforms/compare-strip

```diff
@@ -30,7 +30,7 @@
 # the wrong /usr/lib/pythonXY in the proto area, remove them.
 #
 # We always check PYDIRVP (the default python) and hard-code the "other" to drop.
-<transform file path=$(PYDIRVP)/.*/.*cpython-39.pyc -> drop>
```

Comment:
This looks the wrong way round. I think we should have both lines for now.

Reply:
It looks like it's the right way around to me (and it works), what have I missed?

Reply:
Sorry. Yes you are right. It is the right way round. My mind was stuck on the python magic we do with pyc files in oi-userland. This is a similar but different case. Sorry for the noise.

https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/transforms/autopyc